UNIFY-642-cypress app (#4262)
cypress-io/cypress-documentation: content/guides/references/troubleshooting.md

### Run the Cypress app by itself

Cypress comes with an npm CLI module that parses the arguments, starts the Xvfb

> server (if necessary), and then opens the Cypress App application built on top

Reviewer: ```suggestion
server (if necessary), and then opens the Cypress App built on top
```

### Patch Cypress

Cypress comes with an npm CLI module that parses the arguments, starts the Xvfb

> server (if necessary), and then opens the Cypress App application built on top

Reviewer: ```suggestion
server (if necessary), and then opens the Cypress App built on top
```

You can change anything in the JavaScript code:

<DocsImage src="/img/guides/source-code.png" alt="Source code of the Cypress App in a text editor" ></DocsImage>

> When finished, if necessary, remove the edited application version and reinstall

Reviewer: ```suggestion
When finished, if necessary, remove the edited Cypress App version and reinstall
```
